README.md

```diff
@@ -47,10 +47,11 @@ Santiment API python client.
     - [News](#news)
     - [Ethereum Spent Over Time](#ethereum-spent-over-time)
     - [Token Top Transactions](#token-top-transactions)
-    - [Fetch single metric](#fetch-single-metric)
+    - [Full list of on-chain metrics (including timebounded)](#full-list-of-on-chain-metrics-(including-timebounded))
```

Comment:
I think this is the correct `full-list-of-on-chain-metrics-including-timebounded`, otherwise link is not working.
Should we put this on top @IvanIvanoff  @spiderjako :?

Reply:
Hm, strange, it seemed to work in VSCode. Just changed it up, should be fine now

Reply:
About the changing on top, maybe we should put it above all_projects?